doc/library/config.txt

```diff
@@ -567,6 +567,19 @@ import theano and print the config variable, as in:
 
     A directory with bin/, lib/, include/ folders containing cuda utilities.
 
+.. attribute:: dnn.enabled
+
+  String value: ``auto``, ``True``, ``False``
+
+  Default ``auto``
+
+  If ``auto``, auto detect if cudnn is there, if not, raise an error.
+
+  If ``True``, if we can't use cudnn, raise an error.
```

Comment:
auto, autodetect. If present use it, if not present, don't use it, but don't raise an error.

True, raise an error if no present. I'll update the doc.